### PR TITLE
update docs to reflect new default behavior for kots get config

### PR DESCRIPTION
**Pull request by kcboyle**

### CODE REVIEW

File: content/kots-cli/get/config.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ kubectl kots get config [flags]
...
-| `-h, --help`      |        | help for get apps                                                   |
-| `-n, --namespace` | string | the namespace where the admin console is running _(required)_       |
+| Flag              | Type   | Description                                                                                       |
+| :---------------- | ------ |---------------------------------------------------------------------------------------------------|
+| `--appslug`       | string | The app slug for which the config will be retrieved _(required if more than one app is deployed)_ |
+| `--sequence`      | int    | The app sequence for which the config will be retrieved _(default: latest)_                       |
+| `--decrypt`       | bool   | Decrypt password items within the config                                                          |
+| `-h, --help`      |        | help for get config                                                                               |
+| `-n, --namespace` | string | the namespace where the admin console is running _(required)_                                     |
```
Comment (ajp-io) [inline content/kots-cli/get/config.md:24]: We don't seem to have consistency across this command (or others) about whether the descriptions should start with a capital letter or not. I imagine they should, but I'll let the docs team decide. Even if we update here, other commands appear inconsistent too.

**Review by ajp-io — APPROVED** [review]

**Comment by ajp-io** [comment]
@replicatedhq/replicated-docs Ready for review.

File: content/kots-cli/get/config.md (hunk 1 above)
Comment (paigecalvert) [reply content/kots-cli/get/config.md]: Great catch--and it looks like punctuation isn't consistent either. Added a commit to edit both

**Review by paigecalvert — DISMISSED** [review]
LGTM , @ajp-io! Added a commit with some docs edit--holding off on merging so you have a change to review and comment on the changes

File: content/kots-cli/get/config.md [hunk 2]
```diff
@@ -17,11 +17,11 @@ kubectl kots get config [flags]
...
 | :---------------- | ------ |---------------------------------------------------------------------------------------------------|
-| `--appslug`       | string | The app slug for which the config will be retrieved _(required if more than one app is deployed)_ |
-| `--sequence`      | int    | The app sequence for which the config will be retrieved _(default: latest)_                       |
-| `--decrypt`       | bool   | Decrypt password items within the config                                                          |
-| `-h, --help`      |        | help for get config                                                                               |
-| `-n, --namespace` | string | the namespace where the admin console is running _(required)_                                     |
+| `--appslug`       | string | The app slug of the application from which to retrieve the configValues file. Required if more than one app is deployed. |
+| `--sequence`      | int    | The app sequence from which to retrieve the configValues file. Default: latest.                       |
```
[the remark this answers is not in the document]
Comment (paigecalvert) [reply content/kots-cli/get/config.md]: Sounds good to me--that was the main word I wanted to make sure you could weigh in on :) Committed your suggestion.

**Review by paigecalvert — APPROVED** [review]
Just realized this was held up by another "Approve".

**Pull request merged**